pkg/knuu/knuu.go

```diff
@@ -49,6 +49,8 @@ type Options struct {
 }
 
 func New(ctx context.Context, opts Options) (*Knuu, error) {
+	opts.TestScope = k8s.SanitizeName(opts.TestScope)
```

Comment:
if the testscope is empty, the SanitizeName will panic, but the expected behaviour is to fill it with the default scope.